src/jit/lclvars.cpp

```diff
@@ -133,14 +133,13 @@ void                Compiler::lvaInitTypeRef()
     // Change the compRetNativeType if we are returning a struct by value in a register
     if (!hasRetBuffArg && varTypeIsStruct(info.compRetNativeType))
     {
-#ifdef _TARGET_ARM_
-        // TODO-ARM64-NYI: HFA
+#if FEATURE_MULTIREG_RET && defined(FEATURE_HFA)
```

Comment:
On Arm 64 even structs of size 9-16 bytes are returned in two return registers, right?  For now with this PR we are addressing hfa parameter passing.  You might want to think about how the logic would look like when we have to make changes to support 9-16 byte struct returns.